### ISSUE TITLE
Consider to don't rely on aria-keyshortcuts

### ISSUE THREAD

**Issue by afercia** [swissspidy/preferred-languages#12]
`aria-keyshortcuts` is a new property available in ARIA 1.1, still a Candidate Recommendation.

While it doesn't harm, support is uncertain and it would be wise to don't rely on it as the only way to provide important information.

For reference:
https://www.w3.org/TR/wai-aria-1.1/#aria-keyshortcuts

**Comment by swissspidy** [comment]
Good to know, thanks! What would be a better way to communicate the available keyboard shortcuts?

**Comment by afercia** [comment]
Honestly I don't know :) The spec recommends exposing the shortcuts to all users:
> Authors should provide a way to expose keyboard shortcuts so that all users may discover them, such as through the use of a tooltip.

However, the introduction of tooltips would probably be a bit controversial, even if TinyMCE is already using them.

I'd consider to add some general info about the new UI in the Help tab at the top of the page, that could include also some info about the shortcuts. It won't completely solve the issue, but could help.

I've initially considered aria-labels, for example:
`<button aria-label="Move down (Alt+ArrowDown)" aria-keyshortcuts="Alt+ArrowDown" ... >Move Down</button>`
but that would be redundant and would defeat the purpose of `aria-keyshortcuts` (when it will be supported). A while ago in an accessibility meeting, we were wondering about the need of a new ["inline help system"](https://wordpress.slack.com/archives/accessibility/p1420678367000781) but that would be definitely out of the scope of this project.

Unrelated:
I'd consider to add audible feedback when locales get added, moved up/down or removed, currently there's no feedback. For inspiration, the customizer already does something similar for menu items and widgets, see for example:
'Menu item moved up'
'Menu item moved down'
'Menu item added'
'Menu item deleted'
'Widget moved up'
etc.
This should go in a separate issue, of course :)

**Comment by swissspidy** [comment]
I am still deciding between tooltips and a general help text. @afercia Are there any new suggestions nowadays by the a11y team?

**Comment by swissspidy** [comment]
Unfortunately there's no documentation on [Exposing and Documenting Keyboard Shortcuts](https://www.w3.org/TR/wai-aria-practices-1.1/#kbd_shortcuts_exposing).

**Comment by afercia** [comment]
Yep it's still in the making...
No news from me. While some visually hidden text inside the buttons may help assistive technologies users, that wouldn't communicate the available shortcuts visually. Maybe a general help text is the best option for now?

**Comment by swissspidy** [comment]
Related Gutenberg issues:

https://github.com/WordPress/gutenberg/issues/5420
https://github.com/WordPress/gutenberg/issues/2980
https://github.com/WordPress/gutenberg/issues/71

**Comment by swissspidy** [comment]
I think I am going with tooltips for now until this gets better support in core itself.
